eng/Directory.Build.Common.targets

```diff
@@ -244,14 +244,14 @@
     <!-- *********** Management Client Library Override section ************* -->
   <ItemGroup Condition="'$(IsMgmtLibrary)' == 'true' and '$(IsTestProject)' != 'true'">
 
-    <PackageReference Include="Azure.Core" />
+    <ProjectReference Include="..\..\..\core\Azure.Core\src\Azure.Core.csproj" />
```

Comment:
I am pretty sure we don't want this.  When we ship a mgmt library it should have a package reference to Azure.Core

Reply:
Yes, I agree we shouldn't ship a library with a project reference to Azure.Core.  I believe our release pipelines will prevent this from happening.  The issue here is that the update to PageableHelpers needs types from the unreleased version of Core, and since Azure.ResourceManager uses on PageableHelpers, it needs to be able to use the new types from a package reference.  Our release process is to release Azure.Core first, then update project references back to package references when libraries release.